src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/ShortSegmentCheck.java

```diff
@@ -111,15 +116,17 @@ public boolean validCheckForObject(final AtlasObject object)
     protected Optional<CheckFlag> flag(final AtlasObject object)
     {
         final Edge edge = (Edge) object;
-        final Optional<Node> lowValenceNodes = getConnectedNodesWithValenceLessThan(edge,
-                this.minimumValence);
-        if (lowValenceNodes.isPresent() && !this.isGateLike((Edge) object))
+        final Optional<Triple<Node, Node, Node>> lowValenceNodes = getConnectedNodesWithValenceLessThan(
+                edge, this.minimumValence);
+        if (lowValenceNodes.isPresent() && !this.isGateLike((Edge) object)
+                && !this.isEdgeWithSyntheticBoundaryNode(edge))
```

Comment:
@Bentleysb I think this line is the crux of this patch. This will cause the check to ignore edges that connect to a boundary... correct?

Reply:
Oh, I reread the issue and it seems I thought there was more to the issue than I though. Yes, you are correct, and this makes sense now.